_sass/_syntax.scss

```diff
@@ -10,6 +10,7 @@ div.highlighter-rouge, figure.highlight {
   border-radius: $border-radius;
   background-color: $code-background-color;
   box-shadow: $box-shadow;
+  font-size: $type-size-4
```

Comment:
Should this line have `;` at the end?

Reply:
Yes, it should. It is the last option, so it works _for now_, but clearly just me being an idiot.